tools/ctl/ctl.go

```diff
@@ -368,40 +368,34 @@ var runOne = &cobra.Command{
 		if *ecosystem == "" || *pkg == "" || *version == "" {
 			log.Fatal("ecosystem, package, and version must be provided")
 		}
-		var req *http.Request
-		if mode == benchmark.SmoketestMode {
-			req, err = schema.MakeHTTPRequest(ctx, apiURL.JoinPath("smoketest"), &schema.SmoketestRequest{
-				Ecosystem: rebuild.Ecosystem(*ecosystem),
-				Package:   *pkg,
-				Versions:  []string{*version},
-				Strategy:  strategy,
-				ID:        "runOne",
-			})
-			if err != nil {
-				log.Fatal(err)
-			}
-		} else if mode == benchmark.AttestMode {
-			req, err = schema.MakeHTTPRequest(ctx, apiURL.JoinPath("rebuild"), &schema.RebuildPackageRequest{
-				Ecosystem:        rebuild.Ecosystem(*ecosystem),
-				Package:          *pkg,
-				Version:          *version,
-				StrategyFromRepo: *useStrategyRepo,
-				ID:               "runOne",
-			})
-			if err != nil {
-				log.Fatal(err)
-			}
-		}
-		resp, err := client.Do(req)
+		set := benchmark.PackageSet{
+			Metadata: benchmark.Metadata{
+				Count:   1,
+				Updated: time.Now(),
```

Comment:
I was thinking, maybe this updated time should be some default constant. That way the on-off benchmarks always have the same hash if they're referring to the same target.

Reply:
Yeah you should be able to just omit it, right? To be more confident it'll serialize properly, we could omitempty on the struct (although I'm not sure that's implemented in the form schema).

Reply:
I think the zero value for datetime should probably be good enough? Deleted for now, if it bothers us in the future we can investigate more.